LoopKitUI/DeviceManagerUI.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DeviceManagerUI.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 6/30/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+
+public protocol DeviceManagerUI: DeviceManager {
+    /// An image representing a device configuration after it is set up
+    var smallImage: UIImage? { get }
+}
+
+public extension DeviceManagerUI {
+    /// Default implementations
+    var smallImage: UIImage? { return nil }
```

Comment:
Please remove the default implementation for this. Every PumpManagerUI and CGMManagerUI should implement this. In general, I don't think we should be using protocol default implementations.  It's usually a code smell.

Reply:
Roger that, thanks for pointing it out.  I don't like them, either.  This is leftover from a prior version of the code, will remove.